pkg/http-tunnel/server/addrpool.go

```diff
@@ -36,40 +44,35 @@ func (ap *AddrPool) Init(rang string) error {
 		return fmt.Errorf("Port Range Bad Formated  last %d slower than first %d", ap.last, ap.first)
 	}
 
-	ap.addrMap = make(map[int]*RemoteID, ap.last-ap.first)
-
-	for i := ap.first; i < ap.last; i++ {
-		ap.addrMap[i] = nil
-	}
+	ap.addrMap = make(map[int]*RemoteID)
 
 	return nil
-
 }
 
 func (ap *AddrPool) Acquire(cname string, pname string) (int, error) {
 	ap.mutex.Lock()
 	defer ap.mutex.Unlock()
-	var assigned int
-	assigned = 0
+	var assignedPort int
+	assignedPort = 0
```

Comment:
Why first define `assignedPort` and then initialize it? I think that `assignedPort := 0` is the same thing.